modules/yahoosspBidAdapter.js

```diff
@@ -0,0 +1,570 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, VIDEO } from '../src/mediaTypes.js';
+import * as utils from '../src/utils.js';
```

Comment:
Can you import just the functions needed from `utils` please? We updated all adapters this week so they aren't importing the whole utils file.

Reply:
@ChrisHuie Absolutely - Done!